### PR TITLE
Adding first set of compliance content.

### PR DESCRIPTION
**Pull request by broccolirab**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Dashboards and reports for compliance use-cases. Dashboards and reports use XQL queries to reference the data model and query for values important to regulations. Customer can add filters and modify queries for their specific compliance needs.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by yucohen** [comment]
@broccolirab Any updates here?

**Comment by nkanon** [comment]
Note to @yucohen and me (@nkanon) - Need to add "fromVersion" to all content items so they will only be available from automation version 8.3.0

File: Packs/XSIAMCompliance_HIPAA/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_HIPAA/README.md:8]:
```suggestion
-Produce evidence for audits/regulations.
```

File: Packs/XSIAMCompliance_NIST_800_53/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_NIST_800_53/README.md:7]:
```suggestion
-Reference data that has been mapped to the data model.
```

File: Packs/XSIAMCompliance_NIST_800_53/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_NIST_800_53/README.md:8]:
```suggestion
-Produce evidence for audits/regulations.
```

File: Packs/XSIAMCompliance_NIST_CSF/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_NIST_CSF/README.md:6]:
```suggestion
-Install dashboard and report templates for the following regulations: PCI DSS, HIPAA, NIST CSF, NIST 800-53, NIST 800-171, SOX.
```

File: Packs/XSIAMCompliance_NIST_CSF/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_NIST_CSF/README.md:7]:
```suggestion
-Reference data that has been mapped to the data model.
```

File: Packs/XSIAMCompliance_NIST_CSF/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_NIST_CSF/README.md:8]:
```suggestion
-Produce evidence for audits/regulations.
```

File: Packs/XSIAMCompliance_PCI_DSS/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_PCI_DSS/README.md:6]:
```suggestion
-Install dashboard and report templates for the following regulations: PCI DSS, HIPAA, NIST CSF, NIST 800-53, NIST 800-171, SOX.
```

File: Packs/XSIAMCompliance_PCI_DSS/README.md
Comment (ShirleyDenkberg) [inline Packs/XSIAMCompliance_PCI_DSS/README.md:8]:
```suggestion
-Produce evidence for audits/regulations.
```

**Comment by ShirleyDenkberg** [comment]
@nkanon @yucohen Looks good.  Just missing periods.  Doc review completed.

**Comment by nkanon** [comment]
> @nkanon @yucohen Looks good. Just missing periods. Doc review completed.

Thank you @ShirleyDenkberg

**Comment by nkanon** [comment]
Hi @broccolirab,
Please note that in order to merge this PR we will need you to apply the following updates/changes:
1. Add Widget description to all widgets in all dashboards
2. Add Dashboard description to all dashboards
3. Remove Creator email from all widgets and reports (I can assist with this step)
4. Report names should start with a prefix of the pack name. For example the report of "NIST CSF" needs to look like the following: 
5. Dashboard names should start with a prefix of the pack name

I am here for any help you need.
Thanks!

**Comment by ShirleyDenkberg** [comment]
Added a note about the pack being free for now to all pack readmes.

**Comment by yucohen** [comment]
@broccolirab I'm merging to an internal PR and handle all validation errors there. I'll let you know once we merge it
Thanks for all the effort!!!

**Review by yucohen — APPROVED** [review]

**Pull request merged**
